test_unit.py

```diff
@@ -457,6 +457,109 @@ def test_structured_to_nested_no_value(self):
 
         self._base_structured_to_nested_test(structured_tag, expected_nested_tag)
 
+    """
+    structure to filtered structured test
+    """
+
+    def _base_structured_to_filtered_test(self, structured_tags, expected_filtered_tags, searchTerm):
```

Comment:
The name doesn’t explain much, what the method does. Also, snake case.

```suggestion
    def _base_structured_to_filtered_test(self, structured_tags, expected_filtered_tags, search_term):
```